src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -199,51 +199,52 @@ STATIC int wm_vuldet_insert_agent_data(sqlite3 *db, agent_software *agent, int *
 STATIC void wm_vuldet_free_agent_software(agent_software *agent);
 
 /**
- * @brief Downloads the MSU feed.
+ * @brief Checks whether the MSU table is populated or not.
+ * @param db CVE Database with MSU information.
+ * @return true on success, false otherwise.
+ */
+STATIC bool wm_vuldet_check_enabled_msu(sqlite3 *db);
+
+/**
+ * @brief Downloads a feed (either MSU or ALAS) from Wazuh's own repository.
  * @param update Update structure of the feed being analyzed.
- * @param repo By default is set to NULL; however, if the user specifies another
- * url (tag <url>), this one will be used.
+ * @param repo For MSU feeds, it's by default set to NULL; however, if the user specifies another
+ * URL (tag <url>), this one will be used.
+ * For ALAS, it contains the already fixed URL to the feed's repository.
  * @return 0 if it was successfully downloaded, VU_INV_FEED if it failed,
  *  and VU_NOT_NEED_UPDATE if it's not required.
  */
-STATIC int wm_vuldet_fetch_MSU(update_node *update, char *repo);
+STATIC int wm_vuldet_fetch_wazuh_repo(update_node *update, char *repo);
 
 /**
- * @brief Checks whether the MSU must be updated or not.
+ * @brief Checks whether a feed from Wazuh's repository (either ALAS or MSU) must be updated or not.
  * This is done by checking the metadata.
  * @param update Update structure of the feed being analyzed.
  * @return True if it must be updated, false otherwise.
  */
-STATIC bool wm_vuldet_update_MSU(update_node *update);
+STATIC bool wm_vuldet_update_wazuh_feed(update_node *update);
```

Comment:
The name of this function suggests that the feed will be updated, could we rename it to something like `wm_vuldet_check_feed_metadata`?